scenario_player/main.py

```diff
@@ -144,21 +144,20 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def eth_rpc_option(func):
-    """Decorator for adding '--eth-rpc-endpoint' to subcommands."""
+def environment_option(func):
+    """Decorator for adding '--environment' to subcommands."""
 
     @click.option(
-        "--eth-rpc-endpoint",
-        multiple=False,
-        required=True,
-        help=(
-            '"host:port" address of ethereum JSON-RPC server. '
-            "Accepts a protocol prefix (http:// or https://) with optional port."
-        ),
+        "--environment",
+        default=DEFAULT_ENV_FILE,
+        help="A JSON file containing the settings for Eth-RPC, PFS, "
+        "transport servers, env-type...",
+        show_default=True,
+        type=click.File(),
     )
     @functools.wraps(func)
-    def wrapper(*args, **kwargs):
-        return func(*args, **kwargs)
+    def wrapper(*args, environment: IO, **kwargs):
+        return func(*args, environment=_load_environment(environment), **kwargs)
```

Comment:
FYI: This could be implemented as a click `ParamType` subclass. But this should be fine as well.

Example in Raiden: https://github.com/raiden-network/raiden/blob/07b21ac2bf8a935e26347267b2379e63726e630b/raiden/utils/cli.py#L225-L232